example/index.js

```diff
@@ -14,6 +14,6 @@ var after = new mapboxgl.Map({
   style: 'mapbox://styles/mapbox/dark-v8'
 });
 
-new mapboxgl.Compare(before, after, {
+window.compare = new mapboxgl.Compare(before, after, {
```

Comment:
I'm ok with this (and will defer to @tristen) - but why default to pushing this into the window?

Reply:
You need to place this in window for it to be accessible on the console in the browser dev tools.